thundra/wrappers/web_wrapper_utils.py

```diff
@@ -46,11 +48,14 @@ def start_trace(execution_context, tracer, class_name, domain_name, request):
     execution_context.scope = scope
     execution_context.trace_id = trace_id
 
-    trigger_operation_name = request.get('headers').get(constants.TRIGGER_RESOURCE_NAME_TAG) or \
+    trigger_operation_name = url_rule or request.get('headers').get(constants.TRIGGER_RESOURCE_NAME_TAG) or \
                              request.get('host', '') + normalized_path
-    execution_context.application_resource_name = normalized_path
-    invocation_support.set_agent_tag(constants.HttpTags['HTTP_METHOD'], request.get('method'))
+    execution_context.application_resource_name = url_rule or normalized_path
     invocation_support.set_agent_tag(constants.SpanTags['TRIGGER_OPERATION_NAMES'], [trigger_operation_name])
+    if url_rule:
+        execution_context.trigger_operation_name = url_rule
```

Comment:
Set to `trigger_operation_name` assigned above even if `url_rule` is not exist?
```
execution_context.trigger_operation_name = trigger_operation_name
```

Reply:
Also, can we use the same common logic also for Django? And is it possible to get rid of this logic from there:
```python
if execution_context.scope:
                    execution_context.scope.span.operation_name = request.resolver_match.route
                    execution_context.trigger_operation_name = request.resolver_match.route
                    execution_context.application_resource_name = request.resolver_match.route
                    invocation_support.set_agent_tag(constants.SpanTags['TRIGGER_OPERATION_NAMES'],
                                                     [request.resolver_match.route])
```

Reply:
maybe instead of passing this option inside `request`, we may use additional optional parameter as for ex. `request_route_path` some other name for common usage